maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/utils/WiremockStubs.java

```diff
@@ -148,4 +165,29 @@ public static void stubForUpdateSendToCCLF() {
         stubFor(put(urlMatching(MAAT_API_ASSESSMENT_URL + "/rep-orders/" + TestModelDataBuilder.REP_ID + "/send-to-cclf"))
                 .willReturn(WireMock.ok()));
     }
+
+    public static void stubForDetermineMsgRepDecision(String response) {
+
+        stubFor(post(urlMatching(CCP_URL + "/determine-mags-rep-decision"))
+                .willReturn(WireMock.ok()
+                        .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
+                        .withBody(response)));
+    }
+
+    public static void stubForCreateIncomeEvidence(String response) {
+        stubFor(post(urlMatching(EVIDENCE_URL))
+                .willReturn(WireMock.ok()
+                        .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
+                        .withBody(response)));
+    }
+
+    public static void stubForUpdateFinancialAssessment(String response) {
+        stubFor(put(urlMatching(MAAT_API_ASSESSMENT_URL + "/financial-assessments"))
```

Comment:
I think we need to introduce some new constants for the URLs, there are a ton of duplicates here.